_pages/create_an_account.md

```diff
@@ -1,23 +1,45 @@
 ---
-title: meta.what-is-login.title
-description: meta.what-is-login.description
+title: meta.create-an-account.title
+description: meta.create-an-account.description
 permalink: /create-an-account/
 twitter_card: large
 image: /assets/img/login-gov-600x314.png
 ---
 
 {% capture heading %}
-  {% t what-is-login-page.hero.heading %}
+{% t create-an-account.hero.heading %}
 {% endcapture %}
 
 {% capture text %}
-  {% t what-is-login-page.hero.text %}
+{% t create-an-account.hero.text %}
 {% endcapture %}
 
-{% include hero.html class="what-is-login" heading=heading text=text col_class="sm-col-6" %}
+{% include hero.html class="create-an-account" heading=heading text=text col_class="sm-col-6" %}
 
-<div class="bg-white">
-  <div class="container create-an-account">
-    <div>Create an account: META / HERO / CONTENT TBD</div>
+<div class="create-an-account bg-white grid-container-desktop-lg padding-top-9">
+  <div class="grid-row">
+  <div class="tablet:grid-col-7">
+    <div class="text border-bottom border-blue">{{ site.translations[site.lang]["create-an-account"]["intro"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+    </div>
+    <div class="step-1 step">
+      {{ site.translations[site.lang]["create-an-account"]["step_1"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+    </div>
+    <div class="step-2 step">
+      {{ site.translations[site.lang]["create-an-account"]["step_2"] | replace: 'site.baseurl', site.baseurl | markdownify }}
+    </div>
```

Comment:
Only the third step includes a link where we need to have the `replace` filter. Should those be excluded here?

```suggestion
    <div class="step-1 step">
      {{ site.translations[site.lang]["create-an-account"]["step_1"] | markdownify }}
    </div>
    <div class="step-2 step">
      {{ site.translations[site.lang]["create-an-account"]["step_2"] | markdownify }}
    </div>
```

Reply:
What if we decide to change the content later and include a URL? Then we'd have to change this file _and_ the yaml. I think we should treat this as a template, regardless of the content. Open to discussion.

Reply:
That's true, but the same could be said of any localized text in the site. If it were cause for concern, we could potentially consider applying these sorts of replacements globally as part of some wrapper translation helper.

I don't think it's a big concern one way or the other, but personally I'd err on the side of using filters we know to be needed for the string.